config/prod/shared.yaml

```diff
@@ -0,0 +1,6 @@
+template_path: shared.yaml
```

Comment:
ohh, i forgot we don't want to deploy this multiple times.  what i normally do is create a separate folder called "common",  there an example in agora project, https://github.com/Sage-Bionetworks/Agora-infra/tree/develop/config/common  and  then change travis to run only once, https://github.com/Sage-Bionetworks/Agora-infra/blob/develop/.travis.yml#L28

please follow that pattern.